src/index.js

```diff
@@ -0,0 +1,13 @@
+import {blockingChannel, slidingChannel, droppingChannel} from './factory'
+import Channel from './channel'
+import timeout from './timeout'
+import alts from './alts'
+
+const chan = blockingChannel
+chan.sliding = slidingChannel
+chan.dropping = droppingChannel
+chan.Channel = Channel
+chan.timeout = timeout
+chan.alts = alts
```

Comment:
should be select, considered renaming, but I think it is better to keep go terminology since there is aleady a js library similar to `core/async`.